database/drivers/postgres/parse.go

```diff
@@ -15,9 +15,23 @@ import (
 	"gnorm.org/gnorm/database/drivers/postgres/gnorm/tables"
 )
 
+// PG implements drivers.Driver interface for interacting with postgresql
+// database
+type PG struct {
+}
+
+// Name returns the current driver name which is postgres
+func (db *PG) Name() string {
+	return "postgres"
+}
+
 // Parse reads the postgres schemas for the given schemas and converts them into
 // database.Info structs.
-func Parse(log *log.Logger, conn string, schemaNames []string, filterTables func(schema, table string) bool) (*database.Info, error) {
+func (db *PG) Parse(log *log.Logger, conn string, schemaNames []string, filterTables func(schema, table string) bool) (*database.Info, error) {
```

Comment:
this can be 
```go
func (PG) Parse(
```
we don't use the receiver, so we can drop it.. and it doesn't need to be on a pointer, since it's just an empty struct.